File: linear_search.py
def create_data(n):
    temp = (i for i in range(n))
    return temp
n = 100000
target = n + 1
ma_list = create_data(n)
def linear_search(input_list,target):

    for element in input_list:
        if element == target:
            return True
    return False
print(linear_search(ma_list,target))

File: linear_search.java
import java.util.ArrayList;

public class linear_search {
    public static Boolean search(ArrayList<Integer> input_list, int target){
        for(int curr:input_list){
            if (curr == target){
                return true;
            }
        }


        return false;
    }

    public static ArrayList<Integer> create_d(int n){
        ArrayList<Integer> list = new ArrayList<Integer>();
        for (int i = 0; i<n;i++){
            list.add(i);
        }

        return list;
    }
    public static void main(String[]args){
        int n = 10000000;
        ArrayList<Integer> ma_list = create_d(n);

        boolean res = search(ma_list, n - 1);
        System.out.println(Boolean.toString(res));
    }
}
